docs/sriov-secondary-interface.md

```diff
@@ -0,0 +1,54 @@
+# SR-IOV Secondary Interface Support
+
+This adds SR-IOV support for secondary interfaces in Antrea. API support is
+added to configure Intel 700/800 series NICs in pod namespace. Intel SR-IOV
```

Comment:
I think the PR supports other NICs as well, so maybe we mention "Intel 700/800 series NICs" in section "Update the SR-IOV Network Device Plugin ConfigMap", saying this example works for "Intel 700/800 series NICs"?

Reply:
Sure; will add to the doc to merge later. Thanks.